### PR TITLE
V2: Profile activity

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Builds from #64 

Includes activity for supported proposals.
Does not include comments on proposals, will be added on a separate PR.
 
Also moves the `StakingTokens` and `Wallet` component to Profile page

Screen Shot 2020-09-10 at 11 58 34 AM

### CODE REVIEW

File: src/hooks/useStakes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { addressesEqual } from '../lib/web3-utils'
+
+// TODO: See how to merge it with the implementation done at app-logic
+export function useMyStakes() {
+  const { account } = useWallet()
+  const { proposals } = useAppState()
+
+  return useMemo(
+    () =>
+      proposals.reduce((acc, proposal) => {
```
Comment (rperez89) [inline src/hooks/useStakes.js:14]: i think we already have this , you can do ` const {myStakes} = useAppLogic()`

File: src/providers/Profile.js [hunk 2]
```diff
@@ -45,33 +47,22 @@ function ProfileProvider({ children }) {
...
 
-  const auth = useCallback(async () => {
-    try {
-      const box = await openBoxForAccount(account, ethereum)
-      setBox(box)
-    } catch (err) {
-      console.error(err)
-    }
-  }, [account, ethereum])
+    fetchPrivateData(box)
```
Comment (rperez89) [inline src/providers/Profile.js:64]: let's add the cancel pattern

**Review by rperez89 — COMMENTED** [review]
LGTM some minor comments

File: src/hooks/useStakes.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/hooks/useStakes.js:14]: Ah yeah but the problem there is that we are also using other hooks that we don't want to use here, for example we are not really interested in the `useProposals` hook which will compute and calculate all conviction values. 

That's why i added a TODO there to see how to reuse it but not use the app-logic hooks that are not needed in this case.

Comment (rperez89) [reply src/hooks/useStakes.js:14]: Agree on that! let's do it if possible

**Pull request merged**